src/ipc-handlers.ts

```diff
@@ -106,13 +106,17 @@ export async function getSiteDetails( _event: IpcMainInvokeEvent ): Promise< Sit
 	return mergeSiteDetailsWithRunningDetails( sites );
 }
 
-export function getInstalledApps(): InstalledApps {
+export function getInstalledAppsAndTerminals(): InstalledApps & InstalledTerminals {
 	return {
 		vscode: isInstalled( 'vscode' ),
 		phpstorm: isInstalled( 'phpstorm' ),
 		webstorm: isInstalled( 'webstorm' ),
 		windsurf: isInstalled( 'windsurf' ),
 		cursor: isInstalled( 'cursor' ),
+		terminal: true, // Terminal.app is always available on macOS
```

Comment:
Beside the point of this PR, but I assume this key means `Terminal.app` on macOS and `Command Prompt` on Windows, right? I was mostly thinking of the comment.

Reply:
Yes, we use the same `terminal` key to refer to either default option. We'll show the appropriate label for the app after https://github.com/Automattic/studio/pull/1275 is merged.